src/Bicep.LangServer/Deploy/DeploymentHelper.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Bicep.Core;
+
+namespace Bicep.LanguageServer.Deploy
+{
+    public class DeploymentHelper
+    {
+        public static async Task<string> CreateDeployment(
```

Comment:
Document what this returns

Reply:
Updated as part of this commit:
https://github.com/Azure/bicep/commit/32bed56af070ae3b2c2b494c956f07cf68874667

Thanks!

Reply:
Thanks. FYI, I'm a believer in documenting just what isn't obvious.  No need to fill out all the other params unless they're not obvious from their names or the code (those too often get ignored or out of date).  A function returning an error message string is unusual and so needs documenting.  Thanks!